Tests.Integration.Presentation.Web/app.config

```diff
@@ -100,7 +100,7 @@
       </dependentAssembly>
       <dependentAssembly>
         <assemblyIdentity name="System.Diagnostics.DiagnosticSource" publicKeyToken="cc7b13ffcd2ddd51" culture="neutral" />
-        <bindingRedirect oldVersion="0.0.0.0-6.0.0.2" newVersion="6.0.0.2" />
+        <bindingRedirect oldVersion="0.0.0.0-4.0.3.1" newVersion="4.0.3.1" />
```

Comment:
the diagnostics package can be consolidated across the projects, and possibly even upgraded to the newest version

Reply:
![image](https://github.com/user-attachments/assets/6d48296e-b9c9-4a93-af8b-b64e4f67a473)

Reply:
(notice different versions)

Reply:
this could also impact some package dependencies that will also need to be updated